main.cpp

```diff
@@ -222,48 +237,53 @@ inline string& rtrim(string& s)
 ///////////////////////////////////////////////////////////////////////////
 
 // Arguments:
-//	line - the actual line string without terminating end line
-//	spacesToTabs - Convert spaces to tabs if true, tabs to spaces if false
-inline string& fix_indent(string& line, bool spacesToTabs, unsigned short tabsz)
+//  column - the column where str starts
+//  str - portion of a line consisting of spaces and/or tabs
+//  spacesToTabs - Convert spaces to tabs if true, tabs to spaces if false
+inline string& whitespace_convert(uint8_t column, string& str, bool spacesToTabs, unsigned short tabSizeBefore, unsigned short tabSizeAfter)
```

Comment:
why is `column` the first argument?
It could be more intuitive to use the actual subject (`str`) as the first argument